ui/src/lib/features/k8s/cluster-overview/component.svelte

```diff
@@ -177,15 +184,21 @@
     />
   </div>
 
-  <div class="mt-8 flex flex-col xl:flex-row xl:space-x-4">
-    <div class="w-full mt-4">
-      <div class="p-5 bg-gray-800 rounded-lg overflow-hidden shadow">
-        <CoreServicesWidget {coreServices} {pods} />
+  <div class="flex flex-col xl:flex-row xl:space-x-4">
```

Comment:
```suggestion
  <div class="flex flex-col space-y-8 xl:space-y-0 xl:flex-row xl:space-x-4">
```

This will fix an issue of not having any padding when it switches to rows